src/body/body.rs

```diff
@@ -303,6 +308,11 @@ impl Sender {
             .map_err(|err| err.into_inner().expect("just sent Ok"))
     }
 
+    /// Aborts the request in an abnormal fashion.
```

Comment:
Since this could be a body for a client request or server response, the doc should probably just say it "aborts the body".

Reply:
Fixed